cpp/tests/unary/cast_tests.cpp

```diff
@@ -1003,3 +1003,16 @@ TYPED_TEST(FixedPointTests, Decimal128ToDecimalXXWithLargerScaleAndNullMask)
 
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, result->view());
 }
+
+TEST_F(FixedPointTestSingleType, Int32ToInt64Convert)
```

Comment:
Nit: I'm not sure if we also need to have another test for casting int64 to int32.

Reply:
This is the test that failed from the issue, so it was not covered previously.